### DISCUSSION TITLE
RocketChat hang on login

### DISCUSSION THREAD

**Discussion by bcharbonneaupoly** [RocketChat/Rocket.Chat#31864] [General]
Hello,

We are experiencing issues similar to this one:
https://forums.rocket.chat/t/rocket-chat-docker-server-running-but-page-stuck-on-the-three-dots-loading-page/17979/8

When certain users login, RocketChat is stuck on the three dots.  Opening a second tab on RocketChat temporarely fix the issue on some browser (Chrome, Firefox) but not on other (Edge).

We are running RocketChat 6.6.2 on snap.  Upgrading to 6.6.2 on the edge channel seems too fixed the issue for some users but not all.  Any idea how to fix this?  Thanks!

**Comment by v1d3rm3** [comment]
Here, RocketChat suddenly stop working. The strange thing is that only my user can really open the rocketchat main page (where i can send messages to other users). All users can access login page, but when try to login, "three dots" stay forever. No solution until now.

![3](https://github.com/RocketChat/Rocket.Chat/assets/16581931/99cf336f-aba5-46df-8b90-6d2a3c0a38a3)

Beside, in theses cases, the tab from browser crash with too many requests made.

Just to clarify. We already tried a lot of solutions proposed in some "similar" issues, but no success.
